AltStore/Views/Settings/DevModeView.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  DevModeView.swift
+//  SideStore
+//
+//  Created by naturecodevoid on 2/16/23.
+//  Copyright © 2023 SideStore. All rights reserved.
+//
+
+import SwiftUI
+import LocalConsole
+
+// Yes, we know the password is right here. It's also in CONTRIBUTING.md. It's not supposed to be a secret, just something to hopefully prevent people breaking SideStore with dev mode and then complaining to us.
+let DEV_MODE_PASSWORD = "devmode"
+
+struct DevModePrompt: View {
+    @Binding var isShowingDevModePrompt: Bool
+    @Binding var isShowingDevModeMenu: Bool
+    
+    @State var countdown = 0
+    @State var isShowingPasswordAlert = false
+    @State var isShowingIncorrectPasswordAlert = false
+    @State var password = ""
+    
+    var button: some View {
+        SwiftUI.Button(action: {
```

Comment:
This should probably have the `FilledButtonStyle` applied. The implementation in this branch does not yet support setting the button's `tintColor`, but I have some local changes that implement that functionality for the `FilledButtonStyle`. If you want to change it here, go ahead, but otherwise I would just change it after the changes are merged :)